shared/api/project.go

```diff
@@ -1,5 +1,8 @@
 package api
 
+// DefaultProjectName is the name of the default project that can never be deleted.
+const DefaultProjectName = "default"
```

Comment:
Can we call this ProjectDefaultName so it starts with "Project". In case we ever add something like "ProfileDefaultName" for example. We tend to try prefix the entity type when naming things.

Reply:
Done.